src/main/java/com/aws/greengrass/lifecyclemanager/BrokenService.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.lifecyclemanager;
+
+import com.aws.greengrass.config.Topics;
+import com.aws.greengrass.lifecyclemanager.exceptions.ServiceLoadException;
+
+import java.util.Map;
+
+import static com.aws.greengrass.deployment.bootstrap.BootstrapSuccessCode.REQUEST_RESTART;
+
+public class BrokenService extends GreengrassService {
+    ServiceLoadException serviceLoadException;
+
+    public BrokenService(Topics topics, ServiceLoadException e) {
+        super(topics);
+        this.serviceLoadException = e;
+    }
+
+    @Override
+    protected void install() {
+        serviceErrored(serviceLoadException);
+    }
+
+    @Override
+    public boolean isBootstrapRequired(Map<String, Object> newServiceConfig) {
```

Comment:
why do we want to bootstrap? How does that help us recover? Won't it just be unnecessarily restarting?

Reply:
The intention here is to restart if a deployment tries to fix the broken service, which fails to load previously but has a placeholder instance.